style.scss

```diff
@@ -414,7 +414,7 @@ body >footer {
 					color: $purple-highlight;
 			}
 				
-			&.twitter::before { content: "\e908"; }
+			&.twitter::before { content: "\e909"; }
```

Comment:
Okay, since you've helped me figure out a way to fix the font ordering issue I'll go ahead and fix it. Thanks :)